### ISSUE TITLE
[Feature Request] Use default rundir names in integration tests

### ISSUE THREAD

**Issue by lizziel** [geoschem/geos-chem#1423]
Currently we hard-code names per simulation to use when creating run directories for integration testing. This works well but misses if there is a problem with default run directory names. Using the default names instead of hard-coding them will expand integration testing to also include testing the default naming step of createRunDir.sh.

**Comment by yantosca** [comment]
@lizziel, I will implement this in the draft PR #1565.

**Comment by yantosca** [comment]
@lizziel @msulprizio: Now I remember why I couldn't do this before.  The way I wrote the bash function `create_rundir` (which calls createRunDir.sh), I passed in the directory as an argument because there wasn't an easy way to get the default run directory name out of ./createRunDir.sh:
```bash
function create_rundir() {
    #========================================================================
    # Calls createRunDir.sh with appropriate arguments
    #
    # 1st argument: Commands to forcefeed into createRunDir.sh
    # 2nd argument: Integration test root folder
    # 3rd argument: GEOS-Chem rundir to be created
    # 4th argument: Log file where stdout & stderr will be redirected
    #========================================================================
    cmd=${1}
    root=$(absolute_path ${2})
    runDir=${3}
    log=${4}

    # Create run dir for a short simulation
    printf " ... ${root}/${runDir}\n"
    printf ${cmd} | ./createRunDir.sh >> ${log} 2>&1   # <--- writes commands to a log file
    update_config_files ${root} ${runDir}
}
```
I could tinker with this so that we grep the run directory name from the log file and then use that as the `${runDir} argument to `update_config_files`.  I'll try this.

**Comment by yantosca** [comment]
This has now been added in commit 580a329fad00bea23ad6565d01e5ae0691abe8a2.   Also see PR #1565

**Comment by yantosca** [comment]
Note the default rundir names are now used: `gc_{resolution}_{met}_{simulation etc.}`
![2022_12_16_16_25_28_Greenshot](https://user-images.githubusercontent.com/5880296/208191692-26b61336-2958-4d0c-8941-766b95f72b0d.png)

**Comment by yantosca** [comment]
Working on the GCHP scripts now.

**Comment by yantosca** [comment]
This will go into GEOS-Chem 14.1.1.  See PR #1565 for updates.

**Comment by yantosca** [comment]
We can now close this issue as PR #1565 has been merged into the 14.1.1 development stream.
